Bottom: 74c916967de618742f5df336b6c6aca1160c120d
Top:    89ebfd60338e60917ebaef4e94b7d948d84176dd
Author: Jason Wang <jasowang@redhat.com>
Date:   2015-01-20 13:45:15 +0800

vhost: let vhost_signal() returns whether or not

Signed-off-by: Jason Wang <jasowang@redhat.com>


---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 08ff8a7..fadbe2f 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -1532,9 +1532,6 @@ void vhost_signal(struct vhost_dev *dev, struct vhost_virtqueue *vq)
 	int left;
 	ktime_t now;
 
-	if (!vq->call_ctx)
-		return;
-
 	if (can_coalesce) {
 		now = ktime_get();
 		left = vq->coalesce_usecs -
